test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -76,10 +76,12 @@ private void assertCommandBehavior(String inputCommand,
                                       AddressBook expectedAddressBook,
                                       boolean isRelevantPersonsExpected,
                                       List<? extends ReadOnlyPerson> lastShownList) throws Exception {
-
+        
         //Execute the command
         CommandResult r = logic.execute(inputCommand);
 
+        saveFile.save(addressBook);
```

Comment:
Should we be saving for all commands? Maybe consider throwing in a check to test if mutating commands actually save the files properly.